main/main.cpp

```diff
@@ -903,7 +903,10 @@ Error Main::setup2() {
 
 	path_remap->load_remaps();
 
-	if (show_logo) { //boot logo!
+	List<String> args = OS::get_singleton()->get_cmdline_args();
+	bool editor = args.size()>0 && (args.find("-e")>=0 || args.find("-editor")>=0);
```

Comment:
It would probably be better to initialize this `bool` in the code that parses arguments directly no? Also does this bug only affect the editor when run via those parameters? What about running just `godot` and then loading a project from the project manager?